x/ibc-hooks/go.sum

```diff
@@ -148,7 +148,12 @@ github.com/btcsuite/winsvc v1.0.0/go.mod h1:jsenWakMcC0zFBFurPLEAyrnc/teJEM1O46f
 github.com/bwesterb/go-ristretto v1.2.0/go.mod h1:fUIoIZaG73pV5biE2Blr2xEzDoMj7NFEuV9ekS419A0=
 github.com/bytedance/sonic v1.5.0/go.mod h1:ED5hyg4y6t3/9Ku1R6dU/4KyJ48DZ4jPhfY1O2AihPM=
 github.com/bytedance/sonic v1.10.0-rc/go.mod h1:ElCzW+ufi8qKqNW0FY314xriJhyJhuoJ3gFZdAHF7NM=
+<<<<<<< HEAD
+github.com/bytedance/sonic v1.10.0 h1:qtNZduETEIWJVIyDl01BeNxur2rW9OwTQ/yBqFRkKEk=
+github.com/bytedance/sonic v1.10.0/go.mod h1:iZcSUejdk5aukTND/Eu/ivjQuEL0Cu9/rf50Hi0u/g4=
+=======
 github.com/bytedance/sonic v1.10.1 h1:7a1wuFXL1cMy7a3f7/VFcEtriuXQnUBhtoVfOZiaysc=
+>>>>>>> main
```

Comment:
I think this needs to be resolved, it is weird that CI is green

Reply:
https://github.com/osmosis-labs/osmosis/issues/6538

Reply:
I will address this. Would like to backport and test this PR this week

Reply:
As I understand it, Matt pulled code and got conflict but didn't resolve it, then pushed it and Github detected those lines as code. Is this a problem of syntax error check from Github?

Reply:
I do not think that it is a github problem, I guess the issue is that current linters used in CI are not detecting conflicts in go.sum files (I manually checked, for reference, that they do detect issues in, for example, go mod files).

Reply:
So if I add a step to check for conflict markers ```
<<<<<<<, =======, >>>>>>>
``` , will this case can be solved?

Reply:
I am not sure at the moment what would be the best way to handle this problem. For now, I think it would be worth it checking semantics of how current linters work, maybe there is some linter that is currently disabled that is capable of catching errors in go.sum.

If you are interested in what linters are currently enabled, I would suggest checking this file in the repo: https://github.com/osmosis-labs/osmosis/blob/main/.golangci.yml

Reply:
just one more thought: if running `go mod tidy` solves this issue (i.e. it overwrites conflicts), we could add some workflow that would automatically run `tidy` for every module in the repository (as a matter of fact, we already have an optimized command for doing that: https://github.com/osmosis-labs/osmosis/pull/6535).

Reply:
If I a command to run tidy in submodules after checkout step in build.yml like
`steps:
      - name: Check out repository code
        uses: actions/checkout@v4
      - name: Run go mod tidy in all submodules
        working-directory: ./
        run: make tidy-workspace`
will it work like when we manually run `make tidy-workspace`?

Reply:
I do not think build is the best place to do this, since `tidy-workspace` might change something in the repository, which is not the purpose of this particular workflow. 

Ideally, we should solve the bug by adding a linter that can catch it, a global `go mod tidy` is a way we should go with if a linter option does not work